schemas/event.yml

```diff
@@ -344,6 +400,22 @@
             The installation event type is used for the subset of events within a category
             that indicate that something was installed.
             A common example is `event.category:package` AND `event.type:installation`.
+        - name: protocol
+          description: >
+            The protocol event type is used for the subset of events within a category that
+            indicate that they contain protocol details or analysis. Generally, network events
+            that contain protocol details will fall into this subcategory. A common example is
+            `event.category:network AND event.type:protocol AND event.type:connection` AND event.type:end'
```

Comment:
Move back tick to end to fix formatting.

```suggestion
            `event.category:network AND event.type:protocol AND event.type:connection AND event.type:end`'
```

Reply:
thanks, fixed.